client/src/pages/Signup.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react'
+import React, { useRef } from "react";
+import { Card } from "../components/Card";
```

Comment:
this feels a lot of work, lets setup absolute / named imports in coming tasks

Reply:
https://www.notion.so/App-Setup-absolute-imports-in-the-app-8f20ac71296f4950a9fe9482c5b14bc1?pvs=4